tests/components/admin_console/plugin_management/__snapshots__/plugin_management.test.jsx.snap

```diff
@@ -230,7 +230,7 @@ exports[`components/PluginManagement should match snapshot, upload disabled 1`]
           className="help-text"
         >
           <FormattedHTMLMessage
-            defaultMessage="Installed plugins on your Mattermost server. Pre-packaged plugins are installed by default, and can be deactivated but not removed."
+            defaultMessage="Installed plugins on your Mattermost server. Pre-packaged plugins are installed by default, and can be disabled but not removed."
```

Comment:
Fixed unit test that failed, thanks @smarthoneybee for pointing me in the right direction :)